build.gradle

```diff
@@ -100,9 +101,8 @@ dependencies {
     implementation deps.kotlin.reflect
     implementation deps.kotlin.coroutines
     implementation deps.logback
-    implementation deps.exposed
-    implementation deps.koin.core
-    implementation deps.koin.coreExt
+    implementation deps.joda
```

Comment:
Had to add joda time, because for now I want to keep this as compatible as possible with Aurora